aptos-move/move-examples/fungible_asset/sources/managed_fungible_asset.move

```diff
@@ -0,0 +1,216 @@
+module fungible_asset::managed_fungible_asset {
```

Comment:
I wonder if we should turn this example into a NoCodeCoin? Fungible source + managed_fungible_source are all extensions so we still need something to call the underlying object creation + their init functions to fully initialize a new coin type. This is what the example here is doing